test/services/scraper/bdgest/release_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class Scraper::Bdgest::ReleaseTest < ActiveSupport::TestCase
+  setup do
+    @month = '01'
+    @year = '2020'
+    fixture = "./test/fixtures/bdgest/releases_#{[@month, @year].join}.html"
+    @content = File.read(fixture)
+    @scraper = Scraper::Bdgest::Release.new(@month, @year)
+  end
+
+  test 'should return an array of books if success' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.